btree/btree.c

```diff
@@ -6521,7 +6525,7 @@ static void ut_basic_kv_oper(void)
 		void                *v_ptr  = &value;
 
 		cred = M0_BE_TX_CB_CREDIT(0, 0, 0);
-		m0_btree_put_credit(tree, &cred, 1);
+		m0_btree_put_credit(tree, 1, -1, -1, &cred);
```

Comment:
You cannot send -1 for key size and value size. You need to send the ksize and vsize values. The caller always knows the size of key and value of the record which he will be putting.